esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -1360,14 +1424,141 @@ def create_profile_plot(self, datasets, short_name):
 
+    def create_1d_profile_plot(self, datasets, short_name):
+        """Create 1D profile plot."""
+        plot_type = '1d_profile'
+        if plot_type not in self.plots:
+            return
+
+        if not datasets:
+            raise ValueError(f"No input data to plot '{plot_type}' given")
+
+        # Get reference dataset if possible
+        ref_dataset = self._get_reference_dataset(datasets, short_name)
+        if ref_dataset is None:
+            logger.info("Plotting %s without reference dataset", plot_type)
+            ref_cube = None
+        else:
+            logger.info("Plotting %s with reference dataset '%s'", plot_type,
+                        self._get_label(ref_dataset))
+            ref_cube = ref_dataset['cube']
+
+        # To do : 1d bias plots to reference?
+        bias = self.plots[plot_type].get('plot_bias', {})
+
+        logger.info("Plotting %s", plot_type)
+        fig = plt.figure(**self.cfg['figure_kwargs'])
+        axes = fig.add_subplot()
+
+        multi_dataset_facets = self._get_multi_dataset_facets(datasets)
+
+        # Plot all datsets in one single figure
+        ancestors = []
+        cubes = {}
+        for dataset in datasets:
+            if dataset == ref_dataset and bias:
+                continue
+
+            ancestors.append(dataset['filename'])
+            cube = dataset['cube']
+            cubes[self._get_label(dataset)] = cube
+            self._check_cube_dimensions(cube, plot_type)
+
+            # Plot 1D profile
+            plot_kwargs = self._get_plot_kwargs(plot_type, dataset)
+            plot_kwargs['axes'] = axes
+
+            # apply difference
+            if bias:
+                if ref_cube:
+                    cube, multi_dataset_facets = (
+                        self._compute_bias(cube, ref_cube, bias,
+                                           multi_dataset_facets)
+                    )
+                else:
+                    logger.info("Plotting of differences not possible."
+                                " Reference is missing.")
+
+            iris.plot.plot(cube, **plot_kwargs)
+
+        # Default plot appearance
+        axes.set_title(multi_dataset_facets['long_name'])
+        axes.set_xlabel(f"{short_name} [{multi_dataset_facets['units']}]")
+        z_coord = cube.coord(axis='Z')
+        axes.set_ylabel(f'{z_coord.long_name} [{z_coord.units}]')
+
+        # apply logarithmic axes
+        if self.plots[plot_type]['log_y']:
+            axes.set_yscale('log')
+            axes.get_yaxis().set_major_formatter(
+                FormatStrFormatter('%.1f'))
+
+        if self.plots[plot_type]['log_x']:
+            axes.set_xscale('log')
+
+        # gridlines
+        gridline_kwargs = self._get_gridline_kwargs(plot_type=plot_type)
+        if gridline_kwargs is not False:
+            axes.grid(**gridline_kwargs)
+        # nicer aspect ratio
+        aspect_ratio = self.plots[plot_type].get('aspect_ratio', 3 / 2)
```

Comment:
```suggestion
        aspect_ratio = self.plots[plot_type]['aspect_ratio']
```

If default is set properly in `__init__`